src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -158,6 +167,97 @@ public function enable_widget_copy_paste( $result, $server, $request ) {
 		return $result;
 	}
 
+	/**
+	 * Enable widget copy paste for the Legacy Widgets that we are registering.
+	 * 
+	 * @since 6.11.2
+	 * 
+	 * @param mixed           $result  The result of the rest request.
+	 * @param WP_REST_Request $request The REST request.
+	 * @param string          $route   The route of the request.
+	 * @param array           $handler The handler of the request.
+	 * 
+	 * @return mixed The result of the rest request.
+	 */
+	public function enable_saving_widget_copied( $result, $request, $route, $handler ) {
+		// Bail if result is already set.
+		if ( null !== $result ) {
+			return $result;
+		}
+
+		// Check if this matches our target endpoint.
+		if ( ! str_starts_with( $route, '/wp/v2/widgets' ) ) {
+			return $result;
+		}
+		
+		$widget_type_id = $request->get_param( 'id_base' );
+	
+		// Bail if the widget is not a tribe widget.
+		if ( ! str_starts_with( $widget_type_id, 'tribe-widget-' ) ) {
+			return $result;
+		}
+
+		// Bail if the widget instance is not set.
+		if ( ! isset( $request['instance']['encoded'], $request['instance']['hash'] ) ) {
+			return $result;
+		}
+
+		global $wp_widget_factory;
+
+		$widget_object = $wp_widget_factory->get_widget_object( $widget_type_id );
+
+		// Bail if the widget object is not found.
+		if ( ! $widget_object ) {
+			return $result;
+		}
+
+		// Set the new instance.
+		$new_instance         = $request['instance'];
+		$serialized_instance  = base64_decode( $request['instance']['encoded'] );
+		$new_instance['hash'] = wp_hash( $serialized_instance );
+
+		// Override the instance.
+		$request->set_param( 'instance', $new_instance );
+
+		return $result;
+	}
+
+	/**
+	 * Enable widget copy paste for the Legacy Widgets that we are registering.
+	 * 
+	 * @since 6.11.2
+	 * 
+	 * @param array         $parsed_block The parsed block.
+	 * @param array         $source_block The source block.
+	 * @param WP_Block|null $parent_block The parent block.
+	 * 
+	 * @return array The parsed block.
+	 */
+	public function enable_rendering_widget_copied( $parsed_block, $source_block, $parent_block ) {
+		if ( ! isset( $parsed_block['attrs']['idBase'] ) ) {
+			return $parsed_block;
+		}
+
+		$widget_id = $parsed_block['attrs']['idBase'];
+
+		if ( ! str_starts_with( $widget_id, 'tribe-widget-' ) ) {
+			return $parsed_block;
+		}
+
+		$instance = $parsed_block['attrs']['instance'];
```

Comment:
Do we need a check to ensure `instance` is part of the array before getting it here? Maybe just this:

```suggestion
		$instance = $parsed_block['attrs']['instance'] ?? [];
```